### ISSUE TITLE
Make the themes work for jupyter lab?

### ISSUE THREAD

**Issue by nafizh** [dunovank/jupyter-themes#235]
Incredibly excited with the release of jupyter lab. The only bummer is, it just has two themes at the moment. Is there a way to use the themes here for jupyter lab? 

Btw, long time user of this repository for jupyter notebooks. Many thanks for your work.

**Comment by dunovank** [comment]
Yeah I'd really like to make the switch myself and would definitely like to take my themes with me when I do. Unfortunately it's a completely different framework but I've started looking into it. Will keep everyone up to date about any developments in this direction. (And thanks for your interest!)

**Comment by data83** [comment]
Keep up the good work!

**Comment by qubix12** [comment]
@dunovank How is it going with the jupyterlab porting? Do you think it is possible to reuse the current themes for the new framework?

**Comment by josesho** [comment]
@qubix12 I did a quick and dirty [port](https://github.com/jupyterlab/jupyterlab/issues/3844) of Solarized Dark for JupyterLab; still a WIP but hope it helps!

**Comment by dunovank** [comment]
Awesome! Can't wait to play around with it. Thanks for posting, def keep us updated!

**Comment by Ngiggs** [comment]
@dunovank , can you also look into this if you are developing a JupLab package for themese ?

[https://github.com/jupyter/help/issues/521](url)

I would really pay if you someone could inlcude that package and would not take too much time

**Comment by josesho** [comment]
@Ngiggs I hacked up my own Solarized Light and Dark theme [here](https://github.com/jupyterlab/jupyterlab/issues/3844#issuecomment-407989937). The [parent thread](https://github.com/jupyterlab/jupyterlab/issues/3844) might also be useful in helping you hack up your own theme.

With regards to developing a full-blown JupyterLab theme, you'd need be fairly well-versed in Javascript package development, just a friendly tip!

**Comment by drheatherwalker** [comment]
It looks like you can use [this](https://jupyterlab.readthedocs.io/en/latest/developer/extension_dev.html#themes) to make the theme permanent somehow, rather than having to run the code each time we start up jupyter. Have you tried this?

**Comment by josesho** [comment]
@drheatherwalker I gave it a shot a while back, but ran into some errors. I'm not versed in JavaScript development, and I found that tutorial you linked to a bit opaque for a beginner.... might try again!

**Comment by paulnakroshis** [comment]
I will second the idea that it would be nice to have the theming capable of the notebook in the jupyter lab notebooks. Javascript is unfortunately not my forte. :-(

**Comment by eddy-geek** [comment]
@dunovank is it possible to use `jt` with the Classic Notebook that is built in JupyterLab (and accessible through *Help > Launch Classic Notebook* and the /tree URL)

It does not seem to work for me.
